docs(deposit): add published records (#11)
CERNDocumentServer/docs-cds-rdm: docs/deposit/published-records.md

# Published records

> Published records are finalized records made available in the repository. Once a record is published, it becomes accessible to users.

Reviewer: ```suggestion
Published records are finalized records made available in the repository. Once a record is published, it becomes accessible to other users.
```

Reviewer 2: Also maybe clarify that depending on the restriction settings of the record, it's not just 'users' (which to me implies _authenticated_ users) who can access it, but also any non-authenticated person who access the website

> 

Reviewer: I am thinking to add a small glossary page with some names, to which we will link when describing things.
Here I would like you to add a disclaimer (like a box with differrent color so it stands out)
```
Please note published record is not equal to a public record. Published record means that record is no longer a draft (temporary working version) but finalised record, registered fully in the system.
```

A published record contains finalized metadata and associated files. It also displays additional information such as:

- usage statistics
- versions
- communities
- citation details

## Your records

To see all your records, go to **My Dashboard**.

![My Dashboard header](images/header-my-dashboard.png "My Dashboard header")

The **My Dashboard** page shows all records you have created or contributed to, including published records and drafts.

From this page you can:

- view your published records
- access and continue editing draft records
- monitor the status of your uploads
- search and filter your records
- create new uploads

> Each record entry displays key information such as title, status (`draft`, `published`, `restricted`), and last update date.

Reviewer: we need definitely a glossary page to explain restricted draft and published :) ping @jbenito3 @zzacharo

> Each record entry displays key information such as title, status (`draft`, `published`, `restricted`), and last update date.

Reviewer 2: From the user's perspective, these statuses are not enumerated identifiers. They are depicted with icons, colours, and text

E.g. a green checkmark for `published`

<img width="332" height="99" alt="Image" src="https://github.com/user-attachments/assets/6e4dedd0-8c03-4e42-b040-3d855f8201cb" />

Or a red icon and yellow text for 'in review'

<img width="128" height="60" alt="Image" src="https://github.com/user-attachments/assets/1edbff8b-534f-4c05-8242-a9226318ea71" />

So I don't think we should show them as code-formatted text surrounded by \`backticks\`. We could show them surrounded by 'single quotes' instead for example to demonstrate that they are the names of states

![Uploads page in My Dashboard](images/uploads-page.png "Uploads page in My Dashboard")

## Records shared with you

Records can be shared with you by other users, allowing you to view or collaborate on them.

To access these records, go to **My Dashboard** > **Uploads** and select the **Shared with me** tab.

Shared records may allow different levels of access depending on the permissions granted to you by the record owner.

![Shared with me tab in Uploads](images/uploads-shared-with-me.png "Shared with me tab in Uploads")

## Edit

Draft records can be edited at any time before they are published.

To edit a draft:

1. Open the draft from **My Dashboard** > **Uploads**.
2. Update the metadata or files as needed.
3. Save your changes.

> For published records, only the metadata can be edited.

Reviewer 2: question: are we not allowing the use of the new file modification feature on CDS?

Reviewer: we didn't enable it yet

To edit metadata of a published record:

1. Open the published record you want to modify.
2. Click **Edit**.
3. Update the metadata.
4. Save and publish the changes.

![Edit action on a published record](images/edit-record.png "Edit action on a published record")

If you need to modify or replace files, you must create a new version of the record.

> ## New version

Reviewer 2: Do we want to mention here how DOIs work with new versions? I.e. whether you need to manually request one, and how there is a DOI for the parent record. Or should this go on a different page?

# Published records

Published records are finalized records made available in the repository. Once a record is published, it becomes accessible to other users.

!!! note

    A *published record* is not necessarily a *public record*.  
    Publishing a record means the record is no longer a draft (a temporary working version) but a finalized record registered in the system.  
    Whether the record is publicly accessible depends on its restriction settings.

A published record contains finalized metadata and associated files. It also displays additional information such as:

- usage statistics
- versions
- communities
- citation details

## Your records

> To see all your records, go to **My Dashboard**.

Reviewer 3: I think we can add links as well! WDYT?

Reviewer: yes please add links

![Shared with me tab in Uploads](images/uploads-shared-with-me.png "Shared with me tab in Uploads")

## Edit

Draft records can be edited at any time before they are published.

To edit a draft:

1. Open the draft from **My Dashboard** > **Uploads**.
2. Update the metadata or files as needed.
3. Save your changes.

For published records, only the metadata can be edited.

To edit metadata of a published record:

1. Open the published record you want to modify.
2. Click **Edit**.
3. Update the metadata.
4. Save and publish the changes.

![Edit action on a published record](images/edit-record.png "Edit action on a published record")

If you need to modify or replace files, you must create a new version of the record.

## New version

To modify the files of a published record, create a new version.

To create a new version:

1. Open the published record.
2. Click **New version**.
3. A draft copy of the record is created.
4. Update the metadata or files as needed.
5. Publish the draft to release the new version.

![New version button on a record](images/new-version-button.png "New version button on a record")

CDS keeps a version history, so all previous versions remain accessible.

> All versions are linked together, allowing users to navigate between older and newer versions of the same record.

Reviewer 3: It seemed odd to read from 3rd person perspective speaking about "users" given that in other places we say "you"
```suggestion
All versions of a record are linked together, allowing you to navigate between older and newer versions.
```

Also, can we mention about searching for all versions? The toggle on the search page. Users has dome some doubts about it
